jigasi/rootfs/etc/cont-init.d/10-config

```diff
@@ -51,3 +51,10 @@ if [[ $ENABLE_TRANSCRIPTIONS -eq 1 || $ENABLE_TRANSCRIPTIONS == "true" ]]; then
         }' \
         > /config/key.json
 fi
+
+# check that no conflicting authentications are configured
+if [[ $ENABLE_AUTH -eq 1 && $JIGASI_TRANSCRIBER_AUTHENTICATION -eq 1 ]]; then
```

Comment:
Yeah nobody will understand this. Let's make it work authenticated always, like a recorder please.

Reply:
Agree, not actually intuitive.